hotosm/tasking-manager
# Add dialog to notify user of new version

**How to test:**

- Run `yarn build`
- Serve it with `serve -s build` (you can define a port with `-p`)
- Access the address on your browser
- Verify if you see a message in the bottom-left corner of the page asking you to update. If it doesn't show on the first time, build again and try another time, but always run the server on the same port.
